future-apps/actor/contracts/SignatureValidator.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity 0.4.24;
+
+// From github.com/DexyProject/protocol
+// This should probably be moved into aOS: https://github.com/aragon/aragonOS/pull/442
+
+
+library SignatureValidator {
+    // Maybe use 0x's IDs
```

Comment:
Yes, I think this makes sense, especially in regards to their `ILLEGAL` and `INVALID` types. We should also check the length using the same enum trick they do.